backend/src/main/resources/application.properties

```diff
@@ -1 +1,6 @@
 logging.level.org.springframework.web: DEBUG
+spring.datasource.url=jdbc:postgresql://localhost:5432/
```

Comment:
bitte irgendwo in die readme den command zum starten des dockercontainers aufnehen